flink-cdc-migration-tests/flink-cdc-migration-testcases/pom.xml

```diff
@@ -22,7 +22,7 @@ limitations under the License.
     <parent>
         <groupId>org.apache.flink</groupId>
         <artifactId>flink-cdc-migration-tests</artifactId>
-        <version>${revision}</version>
```

Comment:
I think we should keep using ${revision} instead of 3.3-SNAPSHOT. I also made this mistake in my PR 🙏